### PR TITLE
fix: fix manager_test & manager

### PR DESCRIPTION
**Pull request by baojinri**
# Which issue does this PR close?

Closes #

# Rationale for this change
 existed bug in manager_test & manager
 

# What changes are included in this PR?
fix manager_test & add clientv3 in manager

# Are there any user-facing changes?
none

# How does this change test
exited UT

### CODE REVIEW

File: server/cluster/manager_test.go [hunk 1]
```diff
@@ -190,6 +191,12 @@ func testDropTable(ctx context.Context, re *require.Assertions, manager Manager,
...
 
 func testAllocTableIDWithMultiThread(ctx context.Context, re *require.Assertions, manager Manager, clusterName string, tableID uint64) {
-	go testAllocTableID(ctx, re, manager, node1, clusterName, defaultSchema, table1, tableID)
+	wg := sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		testAllocTableID(ctx, re, manager, node1, clusterName, defaultSchema, table1, tableID)
+		wg.Done()
+	}()
 	testAllocTableID(ctx, re, manager, node2, clusterName, defaultSchema, table1, tableID)
```
Comment (ShiKaiWi) [inline server/cluster/manager_test.go:200]: Only two goroutines are not enough for multi-thread testing. How about starting a dozen of goroutines to do the allocating table id test?

File: server/cluster/manager_test.go [hunk 2]
```diff
@@ -30,15 +31,16 @@ const (
...
-	tableID2                 uint64 = 2
-	tableID3                 uint64 = 3
-	tableID4                 uint64 = 4
+	defaultSchemaID          uint32 = 0
+	tableID1                 uint64 = 0
+	tableID2                 uint64 = 1
+	tableID3                 uint64 = 2
+	tableID4                 uint64 = 3
 	testRootPath                    = "/rootPath"
 	num1                            = 0
```
Comment (chunshao90) [inline server/cluster/manager_test.go:40]: `num1`,`num2` has no meaning, and makes no sense.

File: server/cluster/manager_test.go [hunk 3]
```diff
@@ -190,6 +192,15 @@ func testDropTable(ctx context.Context, re *require.Assertions, manager Manager,
 }
 
 func testAllocTableIDWithMultiThread(ctx context.Context, re *require.Assertions, manager Manager, clusterName string, tableID uint64) {
-	go testAllocTableID(ctx, re, manager, node1, clusterName, defaultSchema, table1, tableID)
+	wg := sync.WaitGroup{}
+	wg.Add(defaultThreadNum)
+	for i := 0; i < defaultThreadNum; i++ {
+		go func() {
+			testAllocTableID(ctx, re, manager, node1, clusterName, defaultSchema, table1, tableID)
+			wg.Done()
```
Comment (chunshao90) [inline server/cluster/manager_test.go:200]: Use `defer wg.Done()` at the beginning of func。

File: server/cluster/manager_test.go [hunk 4]
```diff
@@ -190,6 +192,15 @@ func testDropTable(ctx context.Context, re *require.Assertions, manager Manager,
 }
 
 func testAllocTableIDWithMultiThread(ctx context.Context, re *require.Assertions, manager Manager, clusterName string, tableID uint64) {
-	go testAllocTableID(ctx, re, manager, node1, clusterName, defaultSchema, table1, tableID)
+	wg := sync.WaitGroup{}
+	wg.Add(defaultThreadNum)
+	for i := 0; i < defaultThreadNum; i++ {
+		go func() {
```
Comment (chunshao90) [inline server/cluster/manager_test.go:198]: Use `wg.Add(1)`.

**Comment by chunshao90** [comment]
Goroutines in `testCluster` need to use waitgroup to finish running correctly.

**Review by ShiKaiWi — APPROVED** [review]

**Review by chunshao90 — APPROVED** [review]
LGTM

**Pull request merged**